oss-internship-2020/pffft/main_pffft_sandboxed.cc

```diff
@@ -64,63 +64,72 @@ void ShowOutput(const char* name, int N, int cplx, float flops, float t0,
       printf("|      n/a   ");
   } else {
     if (flops != -1) {
-      printf("N=%5d, %s %16s : %6.0f MFlops [t=%6.0f ns, %d runs]\n", N,
+      printf("n=%5d, %s %16s : %6.0f MFlops [t=%6.0f ns, %d runs]\n", n,
              (cplx ? "CPLX" : "REAL"), name, mflops,
              (t1 - t0) / 2 / max_iter * 1e9, max_iter);
     }
   }
   fflush(stdout);
 }
 
+absl::Status PffftMain() {
+  PffftSapiSandbox sandbox;
+  SAPI_RETURN_IF_ERROR(sandbox.Init());
+
+  return absl::OkStatus();
+}
+
 int main(int argc, char* argv[]) {
-  
   // Initialize Google's logging library.
   google::InitGoogleLogging(argv[0]);
 
   gflags::ParseCommandLineFlags(&argc, &argv, true);
 
-  // Nvalues is a vector keeping the values by which iterates N, its value
-  // representing the input length. More concrete, N is the number of
+  // kTransformSizes is a vector keeping the values by which iterates n, its value
+  // representing the input length. More concrete, n is the number of
   // data points the caclulus is up to (determinating its accuracy).
   // To show the performance of Fast-Fourier Transformations the program is
-  // testing for various values of N.
-  int Nvalues[] = {64,    96,     128,        160,         192,     256,
+  // testing for various values of n.
+  constexpr int kTransformSizes[] = {64,    96,     128,        160,         192,     256,
                    384,   5 * 96, 512,        5 * 128,     3 * 256, 800,
                    1024,  2048,   2400,       4096,        8192,    9 * 1024,
                    16384, 32768};
-  int i;
 
   LOG(INFO) << "Initializing sandbox...\n";
 
   PffftSapiSandbox sandbox;
   absl::Status init_status = sandbox.Init();
 
-  LOG(INFO) << "Initialization: " << init_status.ToString().c_str() << "\n";
+  if (absl::Status status = PffftMain(); !status.ok()) {
```

Comment:
Put this block below `gflags::ParseCommandLineFlags()` and move the rest of this function into `PffftMain()`.